apiKey.env

```diff
@@ -0,0 +1,12 @@
+# api-output
+# the query string needs to be add in the format described below:
+# ingredientsString = ['1st ingredient','2nd ingredient','3rdingredient'].join(',+') + &number=5'
+# es. &number=5 describes the number of recipes we want to return
+
+# This file needs to ne included in the component where is required with the format below:
+# const Api_Key = process.env.React_App_Food_api_Key 
+# 'React_App_Food_api_Key' is the env variable name
+
+# String build-up: Apistring = 'https://api.spoonacular.com/recipes/findByIngredients?ingredients=' + ingredientsString + Api_Key
+
+React_App_Food_api_Key = '&apiKey=e15954d9258e44c2a72e439a8051ffa9'
```

Comment:
Does this not appear in git?
The reason I ask is because the reason an environment variable is used is so that credentials are not shared on the open web.
This file will be added to git and show the apikey to the world.
I'm not sure this is the desired behaviour.
Can you explain a little more about what is going on as I might be missing something.

Reply:
Once we use it it will be added to the .gitognore. 
Whoever clones the repo will receive instruction on how using it. 
Honestly, I do not know what is going to happen when goes to production because if this file doesn't get uploaded to git how can the app read the key? 
I looked at encrypting the key but I didn't manage to make it work.